packages/devextreme/js/__internal/ui/context_menu/m_menu_base.ts

```diff
@@ -574,12 +574,19 @@ class MenuBase extends HierarchicalCollectionWidget<Properties> {
     const { event, itemData } = actionArgs.args[0];
 
     const $itemElement = this._getItemElementByEventArgs(event);
-    const link = $itemElement && $itemElement.find(`.${ITEM_URL_CLASS}`).get(0);
+    const link = $itemElement && $itemElement.find(`.${ITEM_URL_CLASS}`)[0];
```

Comment:
[minor]
```suggestion
    const link = $itemElement?.find(`.${ITEM_URL_CLASS}`)[0];
```

Reply:
I haven’t made this change because it would need to be cherry‑picked across four versions. We’ll apply it only in the newest versions